setup.py

```diff
@@ -63,13 +63,20 @@
         'sphinx-issues',
         'sphinx_rtd_theme',
         'numpydoc'],
-    'examples': [
-        'jupyter',
-        'matplotlib',
-        'plotly']
+    'examples': ['jupyter']
```

Comment:
Nice catch!

Reply:
Although:
a) do we even need `jupyter` since we require `ipywidgets`?
b) In theory, some examples require `pandas` and `openml`.

Reply:
If I understand correctly, `ipywidgets` only installs `ipykernel` and `ipython` which is not exactly all of `jupyter`. 

Good catch on the pandas and openml deps - adding them now!

Reply:
Great!